pkg/testutil/testdir_test.go

```diff
@@ -124,21 +124,21 @@ func TestApplyDir_Symlinks(t *testing.T) {
 		{"sf1", filepath.Join("d1", "f")},
 	}
 	for _, fromTo := range symlinkTargets {
-		symlink, expected_target := fromTo[0], fromTo[1]
+		symlink, expectedTarget := fromTo[0], fromTo[1]
 		fi, err := os.Lstat(symlink)
 		if err != nil || fi.Mode()&os.ModeSymlink == 0 {
 			t.Errorf("File %v isn't a symlink: err %v\n"+
 				"os.ModeSymlink %032b\n"+
 				"fi.Mode        %032b",
 				symlink, err, os.ModeSymlink, fi.Mode())
 		}
-		actual_target, err := filepath.EvalSymlinks(symlink)
-		if err != nil || expected_target != actual_target {
+		actualTarget, err := filepath.EvalSymlinks(symlink)
```

Comment:
Follow the naming convention elsewhere and call this `gotTarget`.

Reply:
Done